### PR TITLE
WIP: Add Dockerfile

### PR DESCRIPTION
**Pull request by hausdorff**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#     --workdir /wd                 \
+#     ksonnet/ksonnet               \
+#     kubecfg "$@"
+# }
+
+FROM golang:1.8 as kubecfg-builder
+# Keep this in sync with the corresponding ENV in stage 2
+ENV KUBECFG_VERSION v0.5.0
+
+RUN go get github.com/anguslees/kubecfg-1
```
Comment (anguslees) [inline Dockerfile]: heh, I'm flattered but there's no way this should be pulling from my personal fork :P

**Comment by hausdorff** [comment]
To recap, I couldn't figure out how to start CI without issuing a PR, and so when we were hunting down #103, I used this to run some tests in Docker, so that I could determine it wasn't just my machine.

But, now that we've written it, we might as well just polish it up and commit it.

@anguslees what part of this do you want to be exercised by CI? I thought Travis could not build docker images.

**Comment by hausdorff** [comment]
I've added a bunch of goodies to make it easy to deal with the Dockerfile, and I'm going to try to see if we can build from inside Travis to see if it can be exercised by CI.

**Comment by mkmik** [comment]
This PR is stale; probably it makes more sense to add kubecfg to the bitnami application catalog so we don't need to worry with container delivery pipeline here

**Pull request closed without merging**